global_time_series/coupled_global.py

```diff
@@ -23,12 +23,16 @@ def getmoc(dir_in):
     time0=fin['year'][:]
     var0 = fin['mocAtlantic26'][:]
     for iyear in range(np.int(time0[0]),np.int(time0[-1])+1):
-     imon = np.where(time0==iyear)[0]
-     if(len(imon)==12):
-      var=np.append(var,np.mean(var0[imon]))
-      time=np.append(time,iyear)
-     else:
-      print('error in input file :',files[i])
+        if(i>0 and iyear <= time[-1]):
+          #print('the amoc value for year',iyear, 'has been included in the moc time series from another moc file',files[i-1], time[-1], 'Skipping...')
```

Comment:
You need to uncomment this or else Python will fail with `IndentationError: expected an indented block`.

Also, it would be better to format as:
`print("The amoc value for year {} has been included in the moc time series from another moc file {} {}. Skipping...".format(iyear, files[i-1], time[-1]))`